src/net/wooga/jenkins/pipeline/TestHelper.groovy

```diff
@@ -1,6 +1,5 @@
 package net.wooga.jenkins.pipeline
-
-import org.jenkinsci.plugins.pipeline.modeldefinition.Utils
+//import org.jenkinsci.plugins.pipeline.modeldefinition.Utils
```

Comment:
and here

Reply:
This import wasnt being used by the script, and it didn't exist in the refered library. However, I'll remove the comments.

Reply:
Done.